openpathsampling/tests/test_external_engine.py

```diff
@@ -123,33 +122,33 @@ def test_start_stop(self):
         eng = self.fast_engine
         # check that it isn't running yet
         try:
-            assert_equal(eng.proc.is_running(), False)
+            assert not eng.proc.is_running()
         except AttributeError:
-            pass # if eng.proc doesn't exist, then it isn't running
+            pass  # if eng.proc doesn't exist, then it isn't running
 
         # start it; check that it is running
         eng.start(self.template)
-        assert_equal(eng.proc.is_running(), True)
+        assert eng.proc.is_running()
```

Comment:
I arbitrarily decided to test against `truthy` here, not `is True`. Let me know if you want to make this `is True` (and `is False` further down

Reply:
I think truthy is the right way to go. I had a bad habit of automatically using `assert_equal` for everything -- just muscle memory on every assert.